snap/snapcraft.yaml

```diff
@@ -25,19 +25,13 @@ slots:
     udev-tagging:
       - kernel: accel[0-9]
         subsystem: accel
-        attributes:
-          vendor: "0x8086" # Intel
```

Comment:
was this unecessary before?

Reply:
Yeah, I added it in a previous PR and re-tested; however, I think there must have been some old rules lying around because now it breaks the udev tagging on a freshly provisioned system (in other words, I didn't test properly before). 

The reason is that `attributes` creates a `ATTR{vendor}` tag but what we need is `ATTRS{vendor}` (which corresponds to the parent device). See [this MM thread](https://chat.canonical.com/canonical/pl/s1u6oqnjcj8hxywe6acm8zz4sa) if you're interested in seeing more details.